tests/components/openweathermap/test_config_flow.py

```diff
@@ -154,22 +171,24 @@ async def test_form_options(
     assert config_entry.state is ConfigEntryState.LOADED
 
 
-@patch("homeassistant.components.openweathermap.config_flow.OWMClient")
-async def test_form_invalid_api_key(owm_client_mock, hass: HomeAssistant) -> None:
+async def test_form_invalid_api_key(
+    config_flow_owm_client_mock, hass: HomeAssistant
+) -> None:
     """Test that the form is served with no input."""
-    owm_client_mock.return_value = _create_mocked_owm_client(False)
+    config_flow_owm_client_mock.return_value = _create_mocked_owm_client(False)
     result = await hass.config_entries.flow.async_init(
         DOMAIN, context={"source": SOURCE_USER}, data=CONFIG
     )
 
     assert result["errors"] == {"base": "invalid_api_key"}
 
 
-@patch("homeassistant.components.openweathermap.config_flow.OWMClient")
-async def test_form_api_call_error(owm_client_mock, hass: HomeAssistant) -> None:
+async def test_form_api_call_error(
+    config_flow_owm_client_mock, hass: HomeAssistant
```

Comment:
Please make `hass` the first fixture for all tests

Reply:
done